wc-calypso-bridge.php

```diff
@@ -36,7 +36,7 @@
 // TODO Pick a better option name.
 // We can set this during store setup/provisioning so they get the right code loaded.
 // @codingStandardsIgnoreLine
-// update_option( 'is-atomic-ecommerce', true );
+update_option( 'is-atomic-ecommerce', true );
```

Comment:
Was this change intentional, or should we back this out?

Reply:
Whoops- unintentional.  Fixed up.